src/libraries/System.Collections/tests/Generic/CollectionExtensionsTests.cs

```diff
@@ -104,5 +105,25 @@ public void Remove_KeyDoesntExistInIDictionary_ReturnsFalse()
             Assert.False(dictionary.Remove("key", out var value));
             Assert.Equal(default(string), value);
         }
+
+        [Fact]
```

Comment:
Could you add a couple of tests validating behavior when the source collections are null?

Reply:
I added one for IList and one for IDictionary, were you refering to more tests when you wrote "couple"? Those two were the scenarios to test with source collection beeing null I could think of.

Reply:
This should be good, thanks.